### DISCUSSION TITLE
Durchschnittsgeschwindigkeit

### DISCUSSION THREAD

**Discussion by 1maxi2** [devemux86/cruiser#315] [Q&A] [answered]
Wo kann man aktuell die Durchschnittsgeschwindigkeit eingeben ? 
Verwende Brouter offline 
Profil Motorrad. Er errechnet mir für 143 km 11 Stunden. Auch wenn die Route sehr kurvig ist werde ich ungefähr 3 Std. brauchen.
Ist mir in letzter Zeit schon öfters aufgefallen das viel zu hohe Fahrzeiten berechnet werden.

**Comment by devemux86** [comment, marked as answer]
@1maxi2

You can find it under "Settings | Routes | Speeds".
You can also select whether the route duration should be determined from the calculated route or from the average speed.

Sie finden sie unter "Einstellungen | Routen | Geschwindigkeiten".
Sie können auch auswählen, ob die Routendauer aus der berechneten Route oder aus der Durchschnittsgeschwindigkeit ermittelt werden soll.

**Reply by 1maxi2** [reply]
Super, perfekt Danke !
Jetzt ist die errechnete Zeit 2:51 Mit der Einstellung
 " Durchschnittsgeschwindigkeit " wird eine realistische Zeit errechnet.
